### PR TITLE
Feature/rad 923 ( About section almost ready )

### PR DESCRIPTION
**Pull request by rogdevil**

### CODE REVIEW

File: src/components/about/aboutPage1.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    animation: 2.5s anim-lineUp ease-out 1;
+    span.blue {
+      color: var(--primary-100);
+    }
+  }
+
+  .companyDescription {
+    display: grid;
+    grid-template-columns: 1fr 2px 1fr;
+    gap: rem(48px);
```
Comment (prakhar-ira) [inline src/components/about/aboutPage1.scss:24]: won't necessary to given rem to gap.

File: src/components/about/aboutPage5.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
+@import "~sass-rem";
+
+.aboutPage5_parentContainer{
+    color: var(--off-white-100);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: rem(60px);
```
Comment (prakhar-ira) [inline src/components/about/aboutPage5.scss:8]: rem not needed

File: src/components/about/aboutPage5.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    .aboutPage5_joinUs{
+        background: linear-gradient(135deg, #B62BF5 0%, #E52E2C 100%);
+        font-size: rem(64px);
+        font-style: normal;
+        font-weight: 700;
+        line-height: rem(74px);
```
Comment (prakhar-ira) [inline src/components/about/aboutPage5.scss:15]: also for line height rem not needed

File: src/components/about/aboutPage1.scss (hunk 1 above)
Comment (rogdevil) [reply src/components/about/aboutPage1.scss]: done

File: src/components/about/aboutPage5.scss (hunk 2 above)
Comment (rogdevil) [reply src/components/about/aboutPage5.scss]: done

File: src/components/about/aboutPage5.scss (hunk 3 above)
Comment (rogdevil) [reply src/components/about/aboutPage5.scss]: done

**Pull request merged**